addons/advanced_fatigue/functions/fnc_getAnimDuty.sqf

```diff
@@ -0,0 +1,56 @@
+/*
+ * Author: BaerMitUmlaut
+ * Calculates the duty of the current animation.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Animation name <STRING>
+ *
+ * Return Value:
+ * Duty <NUMBER>
+ *
+ * Example:
+ * [player, "AidlPercMstpSlowWrflDnon_G05"] call ace_advanced_fatigue_fnc_getAnimDuty
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+params ["_unit", "_animName"];
+
+private _duty = 1;
+private _animType = _animName select [1, 3];
+
+GVAR(isSwimming) = false;
+
+if (_animType in ["idl", "mov"]) then {
+    switch (_animName select [5, 3]) do {
```

Comment:
does this actually work for every animation? Some of them don't follow the naming sheme. Alternatives?